src/cli.js

```diff
@@ -386,3 +404,12 @@ program.parse(process.argv)
 if (program.rawArgs.length <= 2) {
   program.help()
 }
+
+function errorHandler (e) {
+  if (/404/.test(e.message)) {
+    console.log(chalk.yellow('/!\\') + ' If your space was created under US region, you must provide the region as argument --region us. Otherwise, you can use the default --region eu or omit this flag.')
+  } else {
+    console.log(chalk.red('X') + ' An error occurred when executing the pull-components task: ' + e.message)
```

Comment:
Hey @thiagosaife, as this function is used in a couple of places, you can receive the error message to turn this function more generic